doc/logging_and_error_handling.md

```diff
@@ -42,7 +42,7 @@ An example to configure logging for an application using Datahike:
 
 ## Error Handling
 
-Errors that are caught inside datahike create an `Execution error` that carries similar information like the
+Errors that are caught inside Datahike create an `Execution error` that carries similar information like the
```

Comment:
Is it really an error or is it an exception? Errors in Java mean not-recoverable-kill-the-process-errors. If it is we should think about the wording probably and throw an exception that the user can handle himherself.